p2p/constants.go

```diff
@@ -41,3 +41,12 @@ const Direct BroadcastMethod = "Direct"
 
 // Broadcast defines a broadcast message
 const Broadcast BroadcastMethod = "Broadcast"
+
+// MessageHandlerType defines the type of the message handler
+type MessageHandlerType string
+
+// RegularMessageHandler defines a message handler for the main network
```

Comment:
at least, I won't define the consts here and keep only the type definition

Reply:
completely removed it